src/shortcuts.ts

```diff
@@ -0,0 +1,19 @@
+export interface ShortcutsHook {
+  getShortcutDefinitions(): ShortcutDefinition[]
+}
+
+export interface ShortcutDefinition {
+  id: string // Example: claim-reward
+  name: string // Example: Claim
+  description: string // Example: Claim your reward
+  networks: string[] // Example: ['celo']
+  category?: 'claim' // We'll add more categories later
+  onTrigger: (
```

Comment:
Probably this `onTrigger` can receive an extra parameter to allow shortcuts with user inputs?
But I guess we can add it later.

Reply:
Yes it would need to be extended for uses cases with user inputs.